src/RiveQtQuickItem/riveqsgrhirendernode.h

```diff
@@ -62,9 +64,12 @@ class RiveQSGRHIRenderNode : public RiveQSGRenderNode
     QVector<QRhiResource *> m_cleanupList;
 
     RiveQtRhiRenderer *m_renderer { nullptr };
+    // ToDo: make std::shared_ptr
     QRhiTexture *m_displayBuffer { nullptr };
     QRhiTextureRenderTarget *m_cleanUpTextureTarget { nullptr };
 
     bool m_verticesDirty = true;
     RiveRenderSettings::FillMode m_fillMode;
+
+    PostprocessingSMAA* m_postprocessing;

```

Comment:
initialize as { nullptr } in windows this is actually crashing at startup because the nullptr check is not working without initialization